pool/pool.go

```diff
@@ -195,6 +198,10 @@ func (m MethodIndex) String() string {
 		return "objectHash"
 	case methodObjectSearch:
 		return "objectSearch"
+	case methodContainerAnnounceUsedSpace:
+		return "containerAnnounceUsedSpace"
+	case methodSyncContainerWithNetwork:
+		return "syncContainerWithNetwork"
```

Comment:
These will be removed eventually.

Reply:
Ok, I will remove it from stat

Reply:
I mean from the pool! All of the real client method should be covered. But `syncContainer` is not a proper method at all.

Reply:
Still there, container sync is not a client method.

Reply:
Yes, it uses the client but not it's part. Removed